src/promptflow/promptflow/_sdk/_constants.py

```diff
@@ -20,7 +20,7 @@ def _prepare_home_dir() -> Path:
     """
     from promptflow._utils.logger_utils import LoggerFactory
 
-    logger = LoggerFactory.get_logger(LOGGER_NAME)
+    logger = LoggerFactory.get_logger(LOGGER_NAME, verbosity=logging.WARNING)
```

Comment:
maybe introduce an constant for the Warning here?

Reply:
Updated to use `get_cli_sdk_logger` to avoid wrong usage